### ISSUE TITLE
Incorrect warning about babel/runtime even though latest babel is installed (v1.8)

### ISSUE THREAD

**Issue by igalicodean** [meteor/meteor#10275]
Upgraded my v1.7 meteor app to v1.8.
As instructed, I've also run:
```
 meteor npm install @babel/runtime@latest
```
 everything seems to work (macos mojave), but for some reason I still see the following warning when I start the app's server:
```
The version of @babel/runtime installed in your node_modules directory 
(7.0.0) contains a breaking change which was introduced by 
https://github.com/babel/babel/pull/8266. Please either downgrade by 
running the following command:

  meteor npm install --save-exact @babel/runtime@7.0.0-beta.55

or update to the latest beta version of Meteor 1.7.1, as explained in 
this pull request: https://github.com/meteor/meteor/pull/9942.
```
Tried a clean build, rm -f node_modules, meteor reset, nothing took care of it.

Am I missing something or is this a bug? I can post more resources on demand, pls ask.

**Comment by igalicodean** [comment]
I found this issue was solved. After running another `meteor update` command, the warning disappeared

**Comment by benjamn** [comment]
For future reference, you likely had a Meteor package that registered a compiler plugin that was last built/published using Meteor 1.7.1, so it was still mistakenly using the old `@babel/runtime`. I suspect the package in question has now been republished, which is why `meteor update` solved your problem.
